nsxt/resource_nsxt_policy_transit_gateway_static_routes_test.go

```diff
@@ -0,0 +1,217 @@
+// © Broadcom. All Rights Reserved.
+// The term "Broadcom" refers to Broadcom Inc. and/or its subsidiaries.
+// SPDX-License-Identifier: MPL-2.0
+
+package nsxt
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/terraform"
+)
+
+var accTestPolicyTransitGatewayStaticRoutesCreateAttributes = map[string]string{
+	"display_name":         getAccTestResourceName(),
+	"description":          "terraform created",
+	"enabled_on_secondary": "true",
+	"network":              "2.2.2.0/24",
+	"ip_address":           "3.1.1.1",
+	"admin_distance":       "2",
+}
+
+var accTestPolicyTransitGatewayStaticRoutesUpdateAttributes = map[string]string{
+	"display_name":         getAccTestResourceName(),
+	"description":          "terraform updated",
+	"enabled_on_secondary": "false",
+	"network":              "3.3.3.0/24",
+	"ip_address":           "4.1.1.1",
+	"admin_distance":       "5",
+}
+
+func TestAccResourceNsxtPolicyTransitGatewayStaticRoutes_basic(t *testing.T) {
+	testResourceName := "nsxt_policy_transit_gateway_static_route.test"
+
+	resource.ParallelTest(t, resource.TestCase{
+		PreCheck:  func() { testAccPreCheck(t) },
```

Comment:
This test should only run against `9.1.0` and up, so lets include this precheck: `testAccNSXVersion(t, "9.1.0")`